docs/api/new-ref-doc-JDN.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+---
+
+# Revise a task due date
+
+Updates the `due_date` of an existing task. This can be useful when the task's deadline needs to be adjusted.
+
+## URL
+
+```shell
+{base_url}/tasks/{id}
+```
+
+## Method
+`PATCH`
+
+## Params
```

Comment:
This isn't quite correct.
- `id` is a URL parameter (as shown in the *URL* section)
- `due_date` is a request body property (and should be in the *Request body* section below.

Reply:
Moved 'due-date' to Request Body section.